static/scss/search.scss

```diff
@@ -0,0 +1,35 @@
+.search-textbox {
+  position: relative;
+
+  input.search {
+    padding: 3px 40px 2px 40px;
+    font-size: 22px;
+    font-weight: 600;
+    width: 100%;
+    border-radius: 6.5px 6.5px 0 0;
+    background-color: white;
+    box-sizing: border-box;
+    border-bottom: 2px solid black;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+  }
+
+  .material-icons {
+    position: absolute;
+    z-index: 3;
+    height: 21px;
+    width: 21px;
```

Comment:
I think a `cursor: pointer;` got dropped in the shuffle here. also, it's minor but `.search-icon` and `.clear-icon` both have `top: 6px;`.